packages/node_modules/@webex/plugin-meetings/src/meeting/request.js

```diff
@@ -647,4 +648,23 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       }
     });
   }
+
+  /**
+   * Make a network request to end meeting for all
+   * @param {Object} options
+   * @param {Url} options.locusUrl
+   * @param {String} options.uuid
+   * @returns {Promise}
+   */
+  endMeetingForAll({
+    locusUrl,
+    uuid
+  }) {
+    const uri = `${locusUrl}/${uuid}/${END}`;
```

Comment:
this doesn't look right, surely the locusUrl already contains the id to identify the meeting so appending `/${uuid}/` shouldn't be needed here